gradle/libs.versions.toml

```
@@ -1,20 +1,21 @@
 [versions]
 tasksApp-name = "1.2.2"
-compileSdk = "35"
+compileSdk = "36"
 minSdk = "24"
-targetSdk = "35"
-agp = "8.8.1"
-kotlin = "2.1.10"
-ksp = "2.1.10-1.0.29"
-coroutines = "1.10.1"
-ktor = "3.1.1"
-compose-plugin = "1.7.3"
-compose-m3-adaptive = "1.0.1"
-room = "2.7.0-beta01"
-sqlite = "2.5.0-beta01"
-koin = "4.0.2"
+targetSdk = "36"
+agp = "8.9.2"
+kotlin = "2.1.20"
+ksp = "2.1.20-2.0.0"
+coroutines = "1.10.2"
+ktor = "3.1.2"
+activity-compose = "1.11.0-rc01"
+compose-plugin = "1.8.0-beta02"
+compose-m3-adaptive = "1.1.0-beta02"
+room = "2.7.1"
+sqlite = "2.5.0"
+koin = "4.0.4"
 coil = "3.1.0"
-about-libraries = "11.5.0"
+about-libraries = "11.6.3"
```

Ideally, update to 12.x (either `12.0.1` or `12.1.0-rc01`)
- blocked by https://github.com/mikepenz/AboutLibraries/issues/1144

```suggestion
about-libraries = "12.1.0-rc01"
```

```suggestion
about-libraries = "12.0.1"
```

---

The issue disappear (with a bunch of warnings) by disabling configuration on demand.

```
org.gradle.configureondemand=false
```